enhancements/network/multi-networkpolicy.md

```diff
@@ -0,0 +1,163 @@
+   disagree with what is already in a document, open a new PR with suggested
+   changes.
+
+The `Metadata` section above is intended to support the creation of tooling
+around the enhancement process.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This proposal to bring networkpolicy features for multus interface.
+
+The `Summary` section is incredibly important for producing high quality
+user-focused documentation such as release notes or a development roadmap. It
+should be possible to collect this information before implementation begins in
+order to avoid requiring implementors to split their attention between writing
+release notes and implementing the feature itself. 
+
+A good summary is probably at least a paragraph in length.
+
+## Motivation
+
+Customer wants to make their Kubernetes secure, by filtering traffic from
+unexpected source. Kubernetes provides `NetworkPolicy` feature, however, it
+is only applied to default network interface, not additional network interface
+that is created by Multus CNI. NetworkPolicy for Multus Interface brings
+NetworkPolicy mechanism to additional network interface and it provides customer
+to filter traffic from unexpected source.
+
+### Goals
+
+- Introduce MultiNetworkPolicy CRD
+- Introduce multi-networkpolicy which generates iptables from above CRD
+- Gathering related information (iptables rules in Pod network namespace) by must-gather
+
+### Non-Goals
+
+- Provide eBPF filtering mechanism
+- Provide NetworkPolicy for userspace pod which uses DPDK and so on
+
+## Proposal
+
+### User Stories [optional]
+
+Detail the things that people will be able to do if this is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+
+#### Story 1
+
+Customer can specify how groups of pods are allowed to communicate with each other
+and other network endpoints as Kubernetes NetworkPolicy does.
+
+### Implementation Details/Notes/Constraints [optional]
+
+### Multus
+
+No additional change is required because multi-networkpolicy only consumes 
+network-attachment-definition CRD and it does not change net-attach-def CRD.
+
+### How user enables multi-networkpolicy?
+
+Currently we're going to bring it as Tech Preview first and then GA after a few release,
+hence opt-in style configuration is planned at Tech Preview and make it default at GA.
+Opt-in style configuration will be done at cluster-network-operator as boolean flag
+(such as `useMultiNetworkPolicy: true`).
+
+
+### Risks and Mitigations
+
+## Design Details
```

Comment:
Does the proposal have a dependency on NPWG standard? e.g. including MultiNetworkPolicy in npwg spec.

Reply:
yeah, make sense.